src/dashboards/dynatrace-gcp-function-log-self-monitoring

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
filename is missing .json extension

Reply:
and as this is GCP (not Dynatrace) dashboard, maybe we should put it into separate directory, e.g. `src/dashboards/gcp'? What do you think?

Reply:
There is already dashboard for metric self monitoring, that's why I put it there. For me it's not necessary to create separate directory, maybe someone else could share an opinion.

Reply:
right, there is already another non-Dynatrace dashboard here, I'm fine with keeping it there